src/modules/auth/credentials/credentials.scss

```diff
@@ -89,6 +89,10 @@
 	grid-area: i;
 }
 
+.info-card {
+	min-height: 332px;
```

Comment:
@Samsunggora I dont think this is any good (too specific with some "random" pixel number. This height is too specific. You have to play more with the grid areas where the info card is wrapped into I suppose